tests/t3_fs_api.py

```diff
@@ -108,7 +108,10 @@ async def send(self, arg):
     ctx.server.inodes.destroy()
     await ctx.cache.destroy()
     shutil.rmtree(ctx.cachedir)
-    shutil.rmtree(ctx.backend_dir)
```

Comment:
I don't think that fsck.s3ql should ever remove the backend directory itself. Can we handle this properly in fsck, so that we need no changes here?

Reply:
Not sure the best way to handle this -- when I run mkfs.s3ql, there's a s3ql_metadata and s3ql_s3qno_xxx file in there, so in real use fsck will not remove the backend directory.   I suppose some test cases are testing things involving the cache or database so they are putting nothing into the backend directory? 

I do think it sounds better to have fsck avoid removing the topmost directory rather than dicking with the test cases as I've done.  Maybe I can take advantage of the topmost directory being first or last (I thnk last?) visited by os.walk (not sure on this one though, I don't know that os.walk gives any way to tell that it's on the last item...).  Or simply compare the current path with the backend directory... I'll probably look into doing it that way.